DLTP-854

msuhovec opened the pull request.

jeremyf commented on lib/rof/osf_to_rof.rb:
```diff
@@ -133,7 +132,7 @@ def apply_previous_archived_version_if_applicable(rels_ext)
       this_rof = {}
       this_rof['owner'] = project['owner']
-      this_rof['type'] = 'OsfArchive'
+      this_rof['type'] =  project['package_type']
```
I'm not certain if this should be the archive_type. My understanding of the rof['type'] is that it is expanded to the 'afModel'. The 'afModel' fro both "OSF Project" and "OSF Registration" should be "OsfArchive" (as per the Ruby class in CurateND).

@dbrower could you weigh in?

dbrower replied:
@jeremyf I would agree with you. The `type` field here should be `osfarchive` (case insensitive).

The project/registration should be as `dc:type`.

Again, my memory is hazy. everything should be tested.

msuhovec commented:
I think you're right.

I changed type back to 'OsfArchive'
and added 'dc:type' to the metadata for the 'OSF Project' or 'OSF
Registration'

On Mon, Feb 20, 2017 at 9:15 AM, Jeremy Friesen <notifications@github.com>
wrote:

> *@jeremyf* requested changes on this pull request.
> ------------------------------
>
> In lib/rof/osf_to_rof.rb
> <https://github.com/ndlib/rof/pull/67#discussion_r102025168>:
>
> > @@ -133,7 +132,7 @@ def apply_previous_archived_version_if_applicable(rels_ext)
>      def build_archive_record
>        this_rof = {}
>        this_rof['owner'] = project['owner']
> -      this_rof['type'] = 'OsfArchive'
> +      this_rof['type'] =  project['package_type']
>
> I'm not certain if this should be the archive_type. My understanding of
> the rof['type'] is that it is expanded to the 'afModel'. The 'afModel' fro
> both "OSF Project" and "OSF Registration" should be "OsfArchive" (as per
> the Ruby class in CurateND).
>
> @dbrower <https://github.com/dbrower> could you weigh in?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ndlib/rof/pull/67#pullrequestreview-22767913>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ADNGTKxI8R-lpRldHsLXhxKCbeniC246ks5reZ_ngaJpZM4MEue1>
> .
>
